src/antsibull/data/ansible-community.py.j2

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+# coding: utf-8
+# Author: Mario Lenz <m@riolenz.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2022
+"""The ansible-community CLI program."""
+
+import argparse
+
+
+def main():
+    '''Main entrypoint for the ansible-community CLI program.'''
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--version',
+        action='version',
+        version='Ansible community version {{ version }}',
+        help="show the version of the Ansible community package",
+    )
+    args = parser.parse_args()  # noqa: F841, pylint: disable=unused-variable
+    parser.print_help()
```

Comment:
Does this always print help? I mean, if someone runs `ansible-communty --version`, why should we print the help message here? There's not much help, anyway, since there's only the `--version` at the moment.

Reply:
When `--version` is used, the program exists before `parser.print_help()` is executed (search for `version=` on https://docs.python.org/3/library/argparse.html: `This expects a version= keyword argument in the [add_argument()](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_argument) call, and prints version information and exits when invoked:`).

Reply:
Thanks for explaining @felixfontein! LGTM :-)